helm/platform-health/templates/load_balancer.yaml

```diff
@@ -0,0 +1,19 @@
+{{ if .Values.loadBalancer.enabled }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: platform-health
+  {{- $roleBinding := .Values.roleBinding | default dict }}
+  {{- $labels := mergeOverwrite (.Values.common_labels | default dict) ($roleBinding.labels | default dict) | toYaml | nindent 4 }}
+  labels: {{ $labels }}
+  {{- $annotations := mergeOverwrite (.Values.common_annotations | default dict) ($roleBinding.annotations | default dict) | toYaml | nindent 4 }}
+  annotations: {{ $annotations }}
+spec:
+  selector:
+    app: {{ .Release.Name }}--{{ .Chart.Name }}
+  ports:
+    - port: {{.Values.loadBalancer.port}}
+      targetPort: {{.Values.container.port}}
+  type: LoadBalancer
```

Comment:
Type should be parametrised. `LoadBalancer` doesn't make so much sense when deployed with an Ingress (which is currently the expected model in order to offload TLS), and we would ideally have a templated Ingress too…